src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -1011,6 +983,111 @@ async function fetchRevealBannersTogether() {
   }, 320);
 }
 
+/**
+ * Add the machinery needed to highlight elements in the TOC when scrolling.
+ *
+ */
+async function addTOCScrollSpy() {
+  // Intersection observer options
+  const options = {
+    root: null,
+    rootMargin: `0px 0px -70% 0px`, // Use -70% for the bottom margin to so that intersection events happen in only the top third of the viewport
```

Comment:
```suggestion
    rootMargin: `0px 0px -70% 0px`, // Use -70% for the bottom margin so that intersection events happen in only the top third of the viewport
```